ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandStoreTest.java

```diff
@@ -97,7 +97,7 @@
   private final CommandId commandId =
       new CommandId(CommandId.Type.STREAM, "foo", CommandId.Action.CREATE);
   private final Command command =
-      new Command(statementText, Collections.emptyMap(), Collections.emptyMap());
+      new Command(statementText, false, Collections.emptyMap(), Collections.emptyMap());
```

Comment:
since we don't expect this to ever be false anymore, let's change it to `true` - I think same for a few other places in this PR